pkg/policy/l4.go

```diff
@@ -255,14 +252,12 @@ func (a AuthType) Uint8() uint8 {
 // This must return the strings accepted for api.AuthType
 func (a AuthType) String() string {
 	switch a {
-	case AuthTypeNone:
-		return ""
-	case AuthTypeNull:
-		return "null"
-	case AuthTypeMTLSSpiffe:
-		return "mtls-spiffe"
+	case AuthTypeDisabled:
+		return "disabled"
+	case AuthTypeAuthSpire:
+		return "required"
```

Comment:
string representation of `AuthTypeAuthSpire` should be `spire` and not `required`?